crates/sui-types/src/object.rs

```diff
@@ -554,8 +554,10 @@ impl Display for Owner {
             Self::Immutable => {
                 write!(f, "Immutable")
             }
-            Self::Shared { .. } => {
-                write!(f, "Shared")
+            Self::Shared {
+                initial_shared_version,
+            } => {
+                write!(f, "Shared({})", initial_shared_version.value())
```

Comment:
nit. I know the formatting above isn't _super_ consistent (like that `:` sitting after Object ID), but it might be better to do 
```suggestion
                write!(f, "Shared ( {} )", initial_shared_version.value())
```
Or maybe just change the other ones?

Reply:
I'd be more in favor of just updating this one to have the space -- will do that now :)